archive-gs-factsheet.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+$factsheets = array();
+if ( have_posts() ) {
+	while( have_posts() ) {
+		the_post();
+		$factsheet_data = WSUWP_Graduate_Degree_Programs::get_factsheet_data( get_the_ID() );
+		$factsheet_data['permalink'] = get_the_permalink();
+		if ( ! empty( $factsheet_data['shortname'] ) ) {
+			$factsheets[ $factsheet_data['shortname'] ] = $factsheet_data;
+		} else {
+			$factsheets[ get_the_title() ] = $factsheet_data;
+		}
+	}
+}
+ksort( $factsheets );
+
+get_header();
+
+?>
+
+<main id="wsuwp-main" class="spine-blank-template">
+
+	<?php get_template_part( 'parts/headers' ); ?>
+
+	<section class="single gutter pad-top degreeprograms hero-med">
+		<div class="column one centervertically">
+			<div class="flexwrap left">
+				<h1>Degree Programs</h1>
+			</div>
+		</div>
+	</section>
+
+	<section class="single gutter pad-top degreesearchsection">
+		<div class="column one centervertically">
+			<div class="searchdegreeswrapper"></div>
+		</div>
+	</section>
+
+	<section class="single gutter pad-top whitesection degreelist">
+		<div class="column one">
+
+			<div class="toparea">
+				<div class="pagination"><a class="active" href="#a">A</a> <a href="#b">B</a> <a href="#c">C</a> <a href="#d">D</a> <a href="#e">E</a> <a href="#f">F</a> <a href="#g">G</a> <a href="#h">H</a> <a href="#i">I</a> <a href="#j">J</a> <a href="#k">K</a> <a href="#l">L</a> <a href="#m">M</a> <a href="#n">N</a> <a href="#o">O</a> <a href="#p">P</a> <a href="#q">Q</a> <a href="#r">R</a> <a href="#s">S</a> <a href="#t">T</a> <a href="#u">U</a> <a href="#v">V</a> <a href="#w">W</a> <a href="#x">X</a> <a href="#y">Y</a> <a href="#z">Z</a></div>
+				<div class="key">
+
+					KEY: Doctorate
+					<div class="doctorate exists">D</div>
+					Master's
+					<div class="masters exists">M</div>
+				</div>
+			</div>
+
+			<div class="lettergroup">
+				<a id="a" name="a"></a>
+				<div class="bigletter active">A</div>
+				<div class="bigletterline"></div>
+				<ul>
+				<?php
+				$letter = 'a';
+				foreach( $factsheets as $factsheet_name => $factsheet ) {
```

Comment:
Good catch. I think this would happen if any factsheets started with something other than a-z. Updated in d65e39f.